openslides_votecollector/seating_plan.py

```diff
@@ -5,6 +5,28 @@ def setup_default_plan():
     """
     Adds a default seating plan.
     """
+    limit = 200
```

Comment:
I do not understand these changes. Why do you want to get another default plan. What is with the current default plan?